napari/plugins/_npe2.py

```diff
@@ -501,18 +514,44 @@ def _register_manifest_actions(mf: PluginManifest) -> None:
     app = get_app()
     actions, submenus = _npe2_manifest_to_actions(mf)
     samples_submenu, sample_actions = _get_samples_submenu_actions(mf)
+    widgets_submenu: List[Any] = []
+    widget_actions: List[Any] = []
```

Comment:
Yeah I wasn't sure about this. So this can be an empty list or action/submenu.

I saw:

https://github.com/napari/napari/blob/596d74d00beff2b9ca5ed7bdc07c1e987dff76ca/napari/plugins/_npe2.py#L425-L427

And I think I recall that maybe I cannot type it as e.g., `List[Action]` as the list can be empty (I'll double check this). Maybe there is a better option I don't know about?

Reply:
Okay maybe I mucked up before. I fixed all the `List[Any]`s and mypy seems happy so :tada: